client/main.lua

```diff
@@ -42,7 +41,7 @@ end
 -- Functions
 
 local function LoadAnimation(dict)
-    RequestAnimDict(dict)
+    lib.requestAnimDict(dict)
     while not HasAnimDictLoaded(dict) do Wait(10) end
 end
```

Comment:
Instead of replacing the one native, replace all function calls to LoadAnimation with just `lib.requestAnimDict` as the loop under is not needed because it does it in that function already